config/domain_variations/3DBall_test.yaml

```diff
@@ -0,0 +1,13 @@
+resampling-interval: 5000
```

Comment:
I do not see why we need a train and test file (I see what this is for from a research perspective, but I do not see how it is useful for the game developer)

Reply:
This is intended for researchers. Specifically those looking to reproduce the results in our JAIR paper, or compare to their own method/algorithm.

Reply:
The argument for would be so people can reproduce the JAIR paper. But I do agree they might be confusing to the user and tend towards keeping them in a separate repo just for us/reproducibility rather than in ML-Agents.

Reply:
I agree @ervteng. The problem is that right now all of the example environment code/demos/etc live here. I would recommend splitting it all off once we move the environments themselves into a separate repo.

Reply:
Hmm, if it's for researchers, should we even have it in our trainer codebase? I anticipate developers looking for a sample YAML to use Domain Variations, and getting confused as to the difference between train and test.

Reply:
But it isn't in our "trainer" codebase, it is in our main codebase. And the sampler is a feature of `ml-agents-envs` not the trainer.

Reply:
The config files are consumed by our trainer codebase. I do like the renaming and putting the sampler configs into a separate folder, but I do not like the idea of introducing "research focused" "features" into the trainer codebase.

Reply:
But it isn't part of the trainer codebase :)

Reply:
The configs are right? The renaming to domain variations looks good to me. I kind of agree with Vince tho that we should leave the train/test configs out of the codebase as they're not really a "feature," more a "use case" that we happen to use.